### PR TITLE
tests: addition of test_examples_app.py

### LINKED ISSUE #113: tests: addition of test_example_app.py
The package contains `examples/app.py` example application, but there is no test
for it in `tests/test_example_app.py`.  It should be added.  See existing examples:

- https://github.com/inveniosoftware/invenio-records/blob/master/tests/test_example_app.py
- https://github.com/inveniosoftware/invenio-marc21/blob/master/tests/test_example_app.py

### PR DESCRIPTION
**Pull request by hachreak**
* Adds test_examples_app.py. (closes #113)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: examples/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```
Comment (jirikuncar) [inline examples/requirements.txt]: do we really need this file?

Comment (hachreak) [inline examples/requirements.txt]: A copy/paste from other [requirements.txt](https://github.com/inveniosoftware/invenio-marc21/blob/master/examples/requirements.txt)

Comment (jirikuncar) [inline examples/requirements.txt]: so we can remove it?

Comment (hachreak) [inline examples/requirements.txt]: I have this issue without it:
```
pkg_resources.DistributionNotFound: The 'SQLAlchemy-Continuum>=1.2.1; extra == "versioning"' distribution was not found and is required by invenio-db
```

Comment (jirikuncar) [inline examples/requirements.txt]: https://github.com/inveniosoftware/invenio-files-rest/blob/master/setup.py#L59-L67 should probably include versioning - but just keep the `requirements.txt` for now

**Pull request merged**